src/components/UserProfile/UserProfile.scss

```diff
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+body{
+background-color: #0077B6;
+}
+
+#userProfile {

```

Comment:
why is there a UserProfile.scss file in this PR I think this should be in the userprofile PR not here